src/main/java/com/arpnetworking/clusteraggregator/ClusterStatusCache.java

```diff
@@ -102,7 +108,17 @@ public void onReceive(final Object message) throws Exception {
         if (!getSender().equals(getSelf())) {
             // Public messages
             if (message instanceof ClusterEvent.CurrentClusterState) {
```

Comment:
Excellent question, as always. This actor I believe checks the state of the akka cluster once every ten seconds by scheduling itself as seen [line 86](https://github.com/ArpNetworking/metrics-cluster-aggregator/blob/fd1fe647de20743a5f14ea4d73b422bdc283cdd3/src/main/java/com/arpnetworking/clusteraggregator/ClusterStatusCache.java#L86) of this file. This state is cached in memory in this actor and served to the status endpoint whenever requested. I assume this is to limit the load the status endpoint is able to put on the `_cluster.sendCurrentClusterState()` call on line 134.